cachito/web/content_manifest.py

```diff
@@ -232,3 +238,205 @@ def generate_icm(self, image_contents=None):
         icm = deepcopy(BASE_ICM)
         icm["image_contents"] = image_contents or []
         return deep_sort_icm(icm)
+
+
+@dataclass(frozen=True)
+class Package:
+    """
+    A package within a content manifest.
+
+    It is used primarily to generate a package URL (purl).
+    """
+
+    name: str
+    type: str
+    version: str
+    dev: bool = False
+    dependencies: list = field(default_factory=list)
+    path: str = None
+    replaces: str = None
```

Comment:
I decided to take away the replaces value. :slightly_smiling_face: 

About importing the `__future__` module, although I think this is a great feature, I saw that it would be the first import to this module on the whole project, so I thought it would be best to leave it aside. If it's something we consider it's worth doing, I'm completely ok to start doing it now, anyways, so let me know.

Reply:
Good point, let's not do it. Come python 3.10, it will be available without the `__future__` import anyway